spec/lib/annotate/annotate_routes_spec.rb

```diff
@@ -235,6 +235,163 @@
             end
           end
         end
+
+        context 'When the result of `rake routes` contains Rake version' do
+          context 'with older Rake versions' do
+            let :rake_routes_result do
+              <<~EOS.chomp
+                (in /bad/line)
```

Comment:
What's supposed to be going on here?

Reply:
The variable defined by `let :rake_routes_result` is just cut-and-pasted.

This variable has been existed before I begin to refactor the test cases.
It seems to be since 69cd69cba2f67fc2362384d61c3ff8c9de45366c.
What this commit means is not recorded as GitHub PR.

Maybe this test does not make sense for result of `rake routes` by recent Rails version, so I'd like to rewrite test cases around this.

Reply:
Gotcha, I'll merge it in for now and feel free to remove in future work.